doc/api/Player_Events.md

```diff
@@ -536,6 +536,48 @@ video track when in directfile mode to avoid that case (this is documented
 in the corresponding APIs).
 </div>
 
+### noPlayableTrack
+
+_payload type_: `Object`
+
+Emitted when no tracks of a given type can be selected for a given period.
+
+The payload is an object with the following properties:
+
+- `trackType` (`"audio" | "video" | "text"`): The type of the track that appears to have
+  no playable options available.
+
+- `period`: (`Object`): The period in which the track is not playable. This object has the
+  following properties:
+
+  - `id`: (`"string"`): The ID of the period.
+
+  - `start`: (`"number"`): The start time of the period.
```

Comment:
First there's a line break between list elements then there's not :O, we must decide!

Reply:
fixed